tests/wpt/metadata/html/semantics/embedded-content/media-elements/playing-the-media-resource/pause-remove-from-document.html.ini

```diff
@@ -1,5 +1,6 @@
 [pause-remove-from-document.html]
   type: testharness
+  expected: TIMEOUT
   [paused state when removing from a document]
-    expected: FAIL
+    expected: TIMEOUT
```

Comment:
Is this a mistake from my part? This tests was timing out every time I ran it.

Reply:
It means that the more of the test now executes than it did before.